go/libkb/upgrade_instructions.go

```diff
@@ -53,12 +53,14 @@ func darwinUpgradeInstructions() {
 	}
 
 	if IsBrewBuild {
-		printUpgradeCommand("brew update && brew upgrade " + packageName)
+		printUpgradeCommand(g, "brew update && brew upgrade "+packageName)
+	} else {
+		g.Log.Warning("  Please download a new version from " + upgradeURI)
```

Comment:
Will the two leading spaces here look weird? Just want to make sure it's intentional.